haproxy/haproxy.cfg

```diff
@@ -17,12 +17,9 @@ option forwardfor
 bind *:80
 http-request add-header X-Forwarded-Proto http
 
-acl host_ui_backend hdr_dom(host) -i jel.ly.fish.local
+acl host_ui_backend hdr_dom(host) -i jel.ly.fish.local livechat.ly.fish.local
```

Comment:
@LucianBuzzo this change is for redirection. Tested locally.

Reply:
@LucianBuzzo Not sure how this works. I assume there is a configuration for redirecting `*.local => *` on production.

Reply:
@karaxuna Have you checked in with @cmfcruz about what we need to do there?